nameko/messaging.py

```diff
@@ -3,22 +3,22 @@
 '''
 from __future__ import absolute_import
 
-import warnings
 from functools import partial
 from logging import getLogger
+import warnings
```

Comment:
Perhaps, although it's going to be another annoying step that frequently gets forgotten and fails the build. Out of order imports are annoying though, and so are casual rearrangements inside other commits.

Could we automatically re-write them on a commit hook? Then the order will always be consistent and any re-arrangement would be made automatically and in its own commit.